src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -42,15 +43,28 @@ const AudioInputProvider: React.FC<Props> = ({
   const selectedInputRef = useRef(selectedAudioInputDevice);
   selectedInputRef.current = selectedAudioInputDevice;
 
-  const replaceDevice = async (device: string): Promise<AudioInputDevice> => {
-    if (onDeviceReplacement) {
```

Comment:
Is this to allow customers to handle their own callback? Do we document it in Storybook? I did not find it (Sorry if I missed it).

Reply:
I think it was added by Jing to handle the device selection when a audio input is a transform device:
https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/sdk-providers-meetingprovider--page

We have documented it as a prop on MeetingProvider.